index.html

```diff
@@ -705,6 +705,18 @@ <h3>Content Distribution Networks</h3>
       </p>
     </section>
 
+<section class="informative">
+<h3>Malicious Issuers</h3>
+<p>
+In general, this specification cannot protect data <a>subject</a> from tracking by a malicious <a>issuer</a>.
```

Comment:
I think we want to open this up to more than "data <a>subject</a>" by referring to tracking credential presentation generally.

```suggestion
In general, the herd privacy protections offered by this specification can be circumvented by malicious <a>issuers</a> and <a>verifiers</a>. Its privacy benefits can only be realized when issuers and verifiers intend to avoid tracking or sharing the presentation of particular credentials.
```